src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -67,7 +67,6 @@ public void Ready(ReadyParams readyParams)
         public void Shutdown()
         {
             DependencyView.Dispose();
-            this.Dispose();
```

Comment:
We should probably move `DependencyView.Dispose();` to `Dispose` rather than keep it here. Feels more natural to do all disposals in `Dispose`.

Reply:
@mmisol Make sense! Will update